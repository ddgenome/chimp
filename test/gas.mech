# a simple gas-phase mechanism
A + B -> k_arrhenius(A1f, E1f) <- k_arrhenius(A1r, E1r) C;
C -> k_arrhenius(A2f, E2f) <- k_arrhenius(A2r, E2r) D;
