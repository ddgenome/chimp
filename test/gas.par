# parameter input file for gas-phase mechanism
A1f	2.0e-1	# mol/Pa^2/m^3/s
E1f	5.2e4	# J/mol
A1r	1.0e4	# mol/Pa/m^3/s
E1r	5.0e4	# J/mol
A2f	5.0e2	# mol/Pa/m^3/s
E2f	4.8e4	# J/mol
A2r	4.9e2	# mol/Pa/m^3/s
E2r	5.0e4	# J/mol
