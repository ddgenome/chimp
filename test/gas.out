# gas_batch
# x	A	B	C	D	steps
0.000000e+00	1.000000e+05	1.000000e+05	0.000000e+00	0.000000e+00	0
1.000019e+00	9.587857e+04	9.587857e+04	4.108877e+03	1.255017e+01	99807
2.000014e+00	9.226677e+04	9.226677e+04	7.687540e+03	4.568594e+01	187807
3.000023e+00	8.911287e+04	8.911287e+04	1.079153e+04	9.559669e+01	265157
4.000027e+00	8.630992e+04	8.630992e+04	1.352743e+04	1.626552e+02	334448
5.000036e+00	8.383782e+04	8.383782e+04	1.591780e+04	2.443763e+02	396105
6.000014e+00	8.166664e+04	8.166664e+04	1.799902e+04	3.343399e+02	450696
7.000037e+00	7.972315e+04	7.972315e+04	1.983660e+04	4.402501e+02	500175
8.000055e+00	7.801487e+04	7.801487e+04	2.143263e+04	5.524975e+02	544128
9.000004e+00	7.646942e+04	7.646942e+04	2.286128e+04	6.693011e+02	584260
1.000004e+01	7.508919e+04	7.508919e+04	2.411493e+04	7.958797e+02	620639
